src/hooks/useContact.js

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react';
+import contactService from '../api/services/contactService';
```

Comment:
ini juga bisa dipanggil seperti ini
```suggestion
import contactService from 'api/services/contactService';
```